tests/ztest/error_hook/src/main.c

```diff
@@ -82,14 +92,32 @@ static void trigger_fault_access(void)
 	printk("b is %d\n", b);
 }
 
-static void trigger_fault_divide_zero(void)
+
+/*
+ * Do not optimize the divide instruction. GCC will generate invalid
+ * opcode exception instruction instead of real divide instruction.
+ * See #33403
```

Comment:
Same comment here about issue number - not a blocker

Reply:
Also removed this issue number. Thanks!